src/transformers/training_args.py

```diff
@@ -627,6 +627,10 @@ class TrainingArguments:
             This will iterate over the entire training dataloader once beforehand,
 
             and will slow down the entire process.
+        neftune_noise_alpha (`Optional[float]`):
+            If not `None`, this will activate NEFTune noise embeddings. This has been proven to drastically improve
+            model performances for instrcution fine-tuning. Check out the original paper here:
+            https://arxiv.org/abs/2310.05914 and the original code here: https://github.com/neelsjain/NEFTune
```

Comment:
```suggestion
            If not `None`, this will activate NEFTune noise embeddings. This can drastically improve
            model performance for instruction fine-tuning. Check out the [original paper](https://arxiv.org/abs/2310.05914)
            and the [original code](https://github.com/neelsjain/NEFTune)
```

Reply:
Let's also include the fact it only is valid for  `PreTrainedModel` and `PeftModel` classes here I think

Reply:
Makes sense! Done!